apps/angular/pipe-easy/src/app/heavy-computation.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'heavComputation',
```

Comment:
There is a typo in the pipe name: it should be `heavyComputation`

Reply:
Yes, thank you!